src/components/Sing/ScoreSequencer.vue

```diff
@@ -1756,7 +1765,7 @@ const contextMenuData = computed<ContextMenuItemData[]>(() => {
   pointer-events: none;
 }
 
-.sequencer-curve {
+.sequencer-volume,.sequencer-pitch {
   grid-row: 2;
   grid-column: 2;
 }
```

Comment:
この形でも良いですが、分けて書くと（重複しますが）少し分かりやすくなるかなと思います…！
```suggestion
.sequencer-pitch {
  grid-row: 2;
  grid-column: 2;
}

.sequencer-volume {
  grid-row: 2;
  grid-column: 2;
}
```